woocommerce/woocommerce-paypal-payments
# Improve error message on certain exceptions (696)

Some more error messages should be improved

### Steps to Test
To mock these error responses, this filter can be used:
```
add_filter('ppcp_request_args', function ($args, $url){
    if (strpos($url,'capture') !== false) {
        $args['headers']['PayPal-Mock-Response'] = '{"mock_application_codes": "INSTRUMENT_DECLINED"}';
    }
    return $args;
}, 10, 2);
```
Change `INSTRUMENT_DECLINED` for the error response you would like to rest. May not work with all errors and only appears to apply for the PayPal gateway and not advanced card payments.
